terraform-aws-vpc/main.tf

```
@@ -96,14 +96,14 @@ resource "aws_route_table" "private" {
 }
 
 resource "aws_route_table_association" "public" {
-  count = 2
+  count          = 2
   subnet_id      = aws_subnet.public[count.index].id
   route_table_id = aws_route_table.public.id
 }
 
 
 resource "aws_route_table_association" "private" {
-  count = 2
+  count          = 2
   subnet_id      = aws_subnet.private[count.index].id
```

Count should be separated form the rest of arguments. Please fix it here and in all other places as well
```suggestion
  count = 2
  
  subnet_id      = aws_subnet.private[count.index].id
```